index.html

```diff
@@ -29,5 +33,122 @@ <h1 class="welcome">Hey there. I'm Pierre barba<br><span class="description"> I'
+                <button type="button"class="buttongreen">See Project</button>
+            </div>
+        </section>
+        <section class="post">
+            <section class="cardInformation">
+                <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li class="myRecentWorkItem">Ruby on rails</li>
+                    <li class="myRecentWorkItem">css</li>
+                    <li class="myRecentWorkItem">JavaScript</li>
+                    <li class="myRecentWorkItem">html</li>
+                </ul>
+                <button type="button"class="buttongreen">See Project</button>
+            </section>
+        </section>
+        <section class="post">
+            <section class="cardInformation">
+                <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li class="myRecentWorkItem">Ruby on rails</li>
+                    <li class="myRecentWorkItem">css</li>
+                    <li class="myRecentWorkItem">JavaScript</li>
+                    <li class="myRecentWorkItem">html</li>
+                </ul>
+                <button type="button"class="buttongreen">See Project</button>
+            </section>
+        </section>
+        <section class="post">
+            <section class="cardInformation">
+                <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li class="myRecentWorkItem">Ruby on rails</li>
+                    <li class="myRecentWorkItem">css</li>
+                    <li class="myRecentWorkItem">JavaScript</li>
+                    <li class="myRecentWorkItem">html</li>
+                </ul>
+                <button type="button"class="buttongreen">See Project</button>
+            </section>
+        </section>
+    </section>
+    <section class="containerAboutMe">
+        <img class="svgImage"src="./illustration 1- about me-mobile.svg" alt="(imgTop)">
+        <h2 class="blueTitle">About me</h2>
+        <p> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button type="button"class="buttonResume">Get My Resume</button>
+    </section>
+    <section class="skillsItem">
+        <section class="aboutMeCard">
+            <img src="./Group.svg" alt="Languages">
+            <h3 class="subtitle">Frameworks</h3>
+            <ul>
+                <li class="aboutMeListItem">Bootstrap</li>
+                <li class="aboutMeListItem">Ruby on rails</li>
+                <li class="aboutMeListItem">RSpec</li>
+                <li class="aboutMeListItem">Capybara</li>
+                <li class="aboutMeListItem">Selenium</li>
+            </ul>
+        </section> <section class="aboutMeCard">
+            <img src="./Group (1).svg" alt="Languages">
+            <h3 class="subtitle">Languages</h3>
```

Comment:
- [OPTIONAL] Please use `ul` and `li` for categories in this section. Languages, Frameworks, and Skills should be listed items. It's a project requirement. As not required by the previous reviewer I consider that optional.